languages/go/exercises/concept/numbers/.meta/example.go

```diff
@@ -0,0 +1,33 @@
+package _meta
+
+// CalculateProductionRatePerHour for the assembly line, taking into account
+// its success rate
+func CalculateProductionRatePerHour(speed int) float64 {
+	const defaultRate = 221.0
+
+	rateForSpeed := defaultRate * float64(speed)
+
+	return rateForSpeed * successRate(speed)
+}
+
+// CalculateProductionRatePerMinute describes how many working items are
+// produced by the asembly line every minute
+func CalculateProductionRatePerMinute(speed int) int {
+	return int(CalculateProductionRatePerHour(speed) / 60)
+}
+
+func successRate(speed int) float64 {
+	if speed == 0 {
+		return 0.0
+	}
+
+	if speed < 5 {
+		return 1.0
+	}
+
+	if speed >= 9 {
+		return 0.77
+	}
+
+	return 0.9
+}
```

Comment:
Hmmm... I forgot we need conditionals here.

Conditionals are supposed to be introduced after `numbers`. But probably it is easier to use some `int` values (in `conditionals` exercise) than it is to use some `if` statements (here) without prior introduction. What do you think?

Reply:
that's a good point, It does make sense to introduce basic types before conditionals to me as well. To that end we could remove the success rate aspect of the exercise and this would still mean that the student would need to think about numeric types and work with maths expressions. Which reminds me that we should talk about that briefly in the introduction as well

Reply:
@DavyJ0nes I wonder: we could have `successRate(speed int) float64` be part of the stub and thus avoid the student having to use `conditionals` here. At the same time he would already get exposed to it a bit by seeing the code...

@ErikSchierboom: What do you think on this?

Reply:
I like this approach of having the `successRate` function stubbed and it keeps parity with the C# exercise as well. Looking at the exercise flow shown in comment below I think we're in a good place for this being one of the early exercises that students will tackle.

Reply:
@DavyJ0nes: Perfect! Can you add the `successRate` function then? Then we can merge this 😃